app.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const bootstrap = require('hof-bootstrap');
+const steps = require('./apps/complaints/steps.js');
+
+bootstrap({
+	views: false,
+	fields: false,
+	routes: [
+		steps
```

Comment:
This isn't really a step. Routes is an array of route objects. Each route object has a `steps` field, but steps are not routes.